.github/workflows/pull_request.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: ['3.9', '3.10', '3.11']
```

Comment:
Since we already moving to py311, can we exclude py39 from this matrix too?

Reply:
Just to verify the backward compatibility with python 3.9, also few QEs and dependent tools(Nailgun/Airgun) still runs things on Python 3.9. 

Also, we validate 3 python versions compatibility at a time so lets keep 3.9!

Reply:
Okay, make sense, soon we can move dependant tools to run on 3.11 too :)

Also, setup.py mentions 3.8, can you update it as well?